tests/acceptance/setup-test.js

```diff
@@ -29,17 +31,7 @@ describe('Acceptance: Setup', function () {
 
         authenticateSession(application);
 
-        visit('/setup/one');
-        andThen(() => {
-            expect(currentURL()).to.equal('/');
-        });
-
-        visit('/setup/two');
-        andThen(() => {
-            expect(currentURL()).to.equal('/');
-        });
-
-        visit('/setup/three');
+        visit('/setup');
```

Comment:
So there's a bug in the Ember andThen helper apparently... I kept running into a failure here where successive `visit` calls in the same `it` call seem to not follow async behavior correctly.

I tried turning on route transition logging, and Ember was attempting to transition to `/` correctly, but the `andThen` was resolving before that transition could be completed. This only happened after the first `visit`/`andThen` combination.

@kevinansfield I know it's bad practice to change this type of assertion in testing, but wasn't sure how else to fix this. I'm not 100% sure why the route change makes it not work 😞